scripts/build/termux_step_massage.sh

```diff
@@ -210,3 +242,368 @@ create_grep_pattern() {
 		echo -n "|$symbol_type$arg"'$'
 	done
 }
+
+# https://android.googlesource.com/platform/bionic/+/main/libc/libc.map.txt
```

Comment:
Why don't we just take this file and put it in version control? Much better IMO than having a bash script just become long version by version. Parsing this shouldnt be tough

Reply:
I have updated this PR to check in the files. Parser maybe excluded from this PR for now.